script/add-grammar

```diff
@@ -43,16 +43,24 @@ def log(msg)
   puts msg if $verbose
 end
 
-def command(*args)
+def command(*args, show_warnings: true)
```

Comment:
**UX nit:** Rename this to
```suggestion
def command(*args, hide_warnings: false)
```

IMHO, it's bad program design to add boolean options that default to true anyway, as it logically contradicts the purpose of a "switch"-type option (`--verbose` makes more sense than `--no-verbose`, for instance).

Reply:
Make sense.

Reply:
Done